### ISSUE TITLE
Better handle near and far camera values depending on camera position

### ISSUE THREAD

**Issue by jailln** [iTowns/itowns#2049]
The near and far plane [are fixed](https://github.com/iTowns/itowns/blob/master/src/Core/Prefab/GlobeView.js#L102) in the `GlobeView` and the `PlanarView`.

1. Make them configurable
2. Update the far distance dynamically depending on the camera position. The far plane is indeed quite far (10 times the earth diameter for the `GlobeView`) to allow zooming out pretty far from the globe or the plane. This value is not adapted when zooming in and tilting the camera: it generates unuseful tiles subdivision and external layers querying and manipulation, which has an impact on performance. Ideally, this should be configurable in some way too.

**Comment by gchoqueux** [comment]
Yes, it's good idea, i started in this branch https://github.com/gchoqueux/itowns/tree/nearFar_2.0
You could use the global bounding box to compute the near and the far.
